### ISSUE TITLE
Styling precedence 

### ISSUE THREAD

**Issue by kannanwisen** [angular/angular#48522]
### Describe the problem that you experienced

I think, the explanation under Styling precedence Heading is inappropriate. 

### Enter the URL of the topic with the problem

https://angular.io/guide/class-binding#styling-precedence

**Comment by alan-agius4** [comment]
It does look like there is a page for this which is not linked from anywhere. 

https://angular.io/guide/style-precedence

**Comment by kannanwisen** [comment]
> It does look like there is a page for this which is not linked from anywhere.
> 
> https://angular.io/guide/style-precedence

Yes, even saw that. Under [styling-precedence](https://angular.io/guide/class-binding#styling-precedence) heading the content is irrelevant. This link has to be provided https://angular.io/guide/style-precedence.

**Comment by bob-watson** [comment]
I'm not sure what you mean by *irrelevant*.

Do you mean to say that the content below the heading is not helpful (which is what it appears to be to me)?

Would linking to [Style Precedence](https://angular.io/guide/style-precedence), or perhaps replacing in the [Class and style binding](https://angular.io/guide/class-binding) with the info from [Style Precedence](https://angular.io/guide/style-precedence) be more helpful?

**Comment by kannanwisen** [comment]
> @kannanwisen Thanks for the feedback, but I'm not sure what you mean by _irrelevant_.
> 
> Do you mean to say that the content below the **[Styling precedence](https://angular.io/guide/class-binding#styling-precedence)** heading is not helpful (which is what it appears to be to me)?

Yes, it is not helpful.

> 
> Would linking to [Style Precedence](https://angular.io/guide/style-precedence), or perhaps replacing in [Class and style binding](https://angular.io/guide/class-binding) with the info from [Style Precedence](https://angular.io/guide/style-precedence) be more helpful?

I feel the below link is better

Replacing in [Class and style binding](https://angular.io/guide/class-binding) with the info from [Style Precedence](https://angular.io/guide/style-precedence) be more helpful?

**Comment by avmaxim** [comment]
@kannanwisen @alan-agius4 @bob-watson From my point of view - we have to keep the docs aligned with the [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) principle where possible, so I encourage avoiding docs duplication in two places: [Class and style binding](https://angular.io/guide/class-binding) and [Style Precedence](https://angular.io/guide/style-precedence). Even more, I suggest removing this section - [## Style Precendece](https://angular.io/guide/class-binding#styling-precedence) - from [Class and style binding](https://angular.io/guide/class-binding) because we already have references to [Style Precedence](https://angular.io/guide/style-precedence) docs in the following sections - [## Binding to multiple styles](https://angular.io/guide/class-binding#binding-to-multiple-styles) and [## Binding to multiple CSS classes](https://angular.io/guide/class-binding#binding-to-multiple-css-classes). I added a bit more info to them to make it more transparent. It will also avoid extra code duplicates and confusion among developers.

I prepared a PR for your review. Feel free to add any comments:
* https://github.com/angular/angular/pull/49435

**Comment by JamesHenry** [comment]
FYI the guide has not been migrated to angular.dev at all, I opened this one to track that: https://github.com/angular/angular/issues/56085

<img width="956" alt="image" src="https://github.com/angular/angular/assets/900523/8cb89a96-5c9c-44ac-b350-c56df3a4b3d4">

**Comment by JeanMeche** [comment]
That doc page is no longer part of our documentation.
